src/replica/replica_stub.cpp

```diff
@@ -335,6 +335,62 @@ void replica_stub::install_perf_counters()
         COUNTER_TYPE_VOLATILE_NUMBER,
         "write size exceed threshold count in the recent period");
 
+    // <- Bulk Load Metrics ->
+
+    _counter_bulk_load_running_count.init_app_counter("eon.replica_stub",
+                                                      "bulk.load.running.count",
+                                                      COUNTER_TYPE_VOLATILE_NUMBER,
+                                                      "current bulk load running count");
+    _counter_bulk_load_downloading_count.init_app_counter("eon.replica_stub",
```

Comment:
This metric is a GAUGE type, right? If it is, `decrease` is missing?

Reply:
https://github.com/XiaoMi/rdsn/blob/95002ff5306b835910a986ddfb636811488e3e3b/include/dsn/perf_counter/perf_counter.h#L36-L43
`COUNTER_TYPE_VOLATILE_NUMBER` is a special type, it needn't decrease the value.